pennylane/ops/qubit/state_preparation.py

```diff
@@ -89,3 +92,33 @@ def decomposition(state, wires):
 
     def adjoint(self):
         raise qml.ops.AdjointError("No adjoint exists for QubitStateVector operations.")
+
+
+class QubitDensityMatrix(Operation):
+    r"""QubitDensityMatrix(state, wires)
+    Prepare subsystems using the given density matrix.
+    If not all the wires are specified, remaining dimension is filled by |0><0|,
+    where leftmost axes has 1 on its element when transposed as |0><0| ⊗ ρ.
+
+    **Details:**
+
+    * Number of wires: Any (the operation can act on any number of wires)
+    * Number of parameters: 1
+    * Gradient recipe: None
+
+    .. note::
+
+        Exception raised if the ``QubitDensityMatrix`` operation is not supported natively on the
+        target device.
+
+    Args:
+        state (array[complex]): a density matrix of size (2**len(wires), 2**len(wires))
+        wires (Sequence[int] or int): the wire(s) the operation acts on
+    """
+    num_params = 1
+    num_wires = AnyWires
+    par_domain = "A"
+    grad_method = None
+
+    def adjoint(self):
+        raise qml.ops.AdjointError("No adjoint exists for QubitDensityMatrix operations.")
```

Comment:
I don't think these lines of code are being tested, could you add a test for this ?

Reply:
OK!